src/systems/eu/eu-component/eu-component-form/eu-component-form-label/eu-component-form-label.twig

```diff
@@ -1,21 +1,38 @@
 {#
   Parameters:
-    - "label" (string or block): label (mandatory)
-    - "for_attribute" (string): ID of the element (optional)
-    - "extra_classes" (string): extra CSS classes to be added
-    - "extra_attributes" (array): extra attributes classes (optional, format: [{ 'name': 'name_of_the_attribute', 'value': 'value_of_the_attribute'}])
+    - "label" (string) (default: '')
+    - "for_attribute" (string) (default: ''): ID of the element
+    - "extra_classes" (string) (default: ''): additional classes to add to the component
+    - "extra_attributes" (array) (default: []): format: [
+        {
+          "name" (string) (default: ''),
+          "value" (string) (default: '')
+        },
+        ...
+      ]
+  Blocks:
+    - "label": main content
 #}
 
+{# Internal properties #}
+
 {% set _css_class = 'ecl-form-label' %}
+{% set _extra_attributes = '' %}
+{% set _label = label|default('') %}
+{% set _for_attribute = for_attribute|default('') %}
+
+{# Internal logic - Process properties #}
 
-{% if extra_classes is defined %}
+{% if extra_classes is defined and extra_classes is not empty %}
   {% set _css_class = _css_class ~ ' ' ~ extra_classes %}
 {% endif %}
 
-{% if extra_attributes is defined %}
+{% if extra_attributes is defined and extra_attributes is not empty and extra_attributes is iterable %}
   {% for attr in extra_attributes %}
     {% set _extra_attributes = _extra_attributes ~ ' ' ~ attr.name ~ '="' ~ attr.value ~ '"' %}
   {% endfor %}
 {% endif %}
 
-<label class="{{ _css_class }}"{{ _extra_attributes|raw }}{{ for_attribute is defined ? ' for="' ~ for_attribute ~ '"'  : '' }}>{% block label label %}</label>
+{# Print the result #}
+
+<label class="{{ _css_class }}"{{ _extra_attributes|raw }}{{ _for_attribute is not empty ? ' for="' ~ _for_attribute ~ '"'  : '' }}>{% block label _label %}</label>
```

Comment:
To me it seems eu-component-form/eu-component-form-label/eu-component-form-label.twig is same as /src/generic/generic-component-form-label/generic-component-form-label.twig should it be `{% include '@ecl/generic-component-form-label' with { system: 'eu' } %}`?